guides/quickstart.md

```diff
@@ -17,15 +17,61 @@ In the following script the basic setup of the database is executed before ZITAD
 
 You can connect to [ZITADEL on localhost:4200](http://localhost:4200) after the frontend compiled  successfully. Initially it takes several minutes to start all containers.
 
+<a name="compose-services"></a>
 ```bash
-COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 \
-&& docker compose -f ./build/local/docker-compose-local.yml --profile backend --profile frontend up
+$ COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker compose -f ./build/local/docker-compose-local.yml --profile backend --profile frontend up --build. Setting everything up takes about 5 minutes.
 ```
 
-For a more detailed guide take a look at the [development guide](./development.md)
+## Developing ZITADEL
+
+Instead of the profiles backend and frondend as described [above](#compose-services), use the profile e2e and detach from containers. Setting everything up takes about 15 minutes. This also initializes data needed by Cypress end-to-end tests. 
+
+<a name="compose-e2e"></a>
+```bash
+$ COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker compose -f ./build/local/docker-compose-local.yml --profile e2e up -d --build
+```
+
+In the meantime, ensure that you have installed the node and npm versions that are known to work for the test suite:
+
+```bash
+$ node --version
+v14.17.6
+
+$ npm --version
+6.14.15
+```
+
+Ater the *e2e-setup* container exited successfully, you are finally ready to launch the Cypress test suite:
+
+```bash
+$ # Change directory to ./console
+$ cd ./console
+
+$ # Install dev dependencies if you haven't done so already
+$ npm install --only development
+
+$ # Run all end-to-end tests
+$ ./cypress.sh open local_local.env
+
+$ # Or open the end-to-end test suite interactively
+$ ./cypress.sh open local_local.env
```

Comment:
Cant we start this with `npm e2e local` it might make more sense to include those commands into the `package.json`